pkgs/by-name/su/subtitlecomposer/package.nix

```diff
@@ -0,0 +1,49 @@
+{ lib
+, fetchFromGitLab
+, cmake
+, extra-cmake-modules
+, ffmpeg
+, openal
+, libsForQt5
+}:
+
+libsForQt5.mkDerivation rec {
+  pname = "subtitlecomposer";
+  version = "0.8.0";
+
+  src = fetchFromGitLab {
+    domain = "invent.kde.org";
+    owner = "multimedia";
+    repo = "subtitlecomposer";
+    rev = "v${version}";
+    sha256 = "sha256-RKS3VTtpxnox0hzessMHmoGPpT+Ho0b3fxtQMGw9OrM=";
+  };
+
+  nativeBuildInputs = [ cmake extra-cmake-modules ];
```

Comment:
I'm surprised to not see `wrapQtAppsHook`

Reply:
Should I add it?